sortinghat/config/settings.py

```diff
@@ -15,6 +15,8 @@
 
 import os
 
+from sortinghat.config.tenants import TENANT_NAMES
```

Comment:
I think it'll be better if we use a json file or similar to get the list of tenants. Also, I would define a new env variable named `MULTI_TENANT_LIST_PATH` so the users can define where the list of tenants are. By default it'll point to `BASE_DIR + "tenants.json"` (or similar). If the variable is define and `MULTI_TENANT` is `True`, multi tenancy will be activated.